package.json

```diff
@@ -14,7 +14,8 @@
     "release": "node script/release",
     "dev": "cross-env NODE_PATH=. nodemon -e js,scss,yml,md server.js",
     "prepack": "check-for-leaks",
-    "prepush": "check-for-leaks"
+    "prepush": "check-for-leaks",
+    "linkschecker": "NODE_PATH=. NODE_ENV=test node scripts/links-checker.js"
```

Comment:
Why do `NODE_PATH` and `NODE_ENV` need to be set?

Reply:
it's so the server would start properly, and without the browsersync stuff.